app/controllers/jobs_controller.rb

```diff
@@ -4,7 +4,7 @@ class JobsController < ApplicationController
   skip_before_action :verify_authenticity_token, :only => :viewed
 
   def index
-    @jobs = Job.published.order("created_at DESC").all
+    @jobs = Job.active.order("created_at DESC")
```

Comment:
We now get active jobs instead of just published jobs now for the job listing.

I removed the `all` here because it was not needed.